### PR TITLE
CODE-196

### PR DESCRIPTION
**Pull request by ArBridgeman**

### CODE REVIEW

File: sous-chef/sous_chef/pantry_list/read_pantry_list.py [hunk 1]
```diff
@@ -25,68 +25,139 @@ def __init__(self, config: DictConfig, gsheets_helper: GsheetsHelper):
...
+                "misspelled_ingredient",
+                "true_ingredient",
+                "label",
+                "replace_factor",
+                "replace_unit",
+            ]
+        ]
+
+        # join all misspelled ingredients with basic pantry list
+        misspelled_list = pd.concat([without_replacement, with_replacement])
```
Comment (ArBridgeman) [inline sous-chef/sous_chef/pantry_list/read_pantry_list.py:140]: When we have a merge with inner, do we want to toss an error afterwards if the shape is less than before the merge?

**Comment by ArBridgeman** [comment]
Would it be useful to add a task to see if we can have our code automatically run itself once a week? With, for instance, the error on merge, this would potentially help us detect pantry list (and in the future other google sheet issues).

File: sous-chef/sous_chef/pantry_list/read_pantry_list.py (hunk 1 above)
Comment (AkagiShigeru) [reply sous-chef/sous_chef/pantry_list/read_pantry_list.py]: Good idea. This has been added.

**Comment by AkagiShigeru** [comment]
> Would it be useful to add a task to see if we can have our code automatically run itself once a week? With, for instance, the error on merge, this would potentially help us detect pantry list (and in the future other google sheet issues).
> 
> https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows
> 
> > The schedule event allows you to trigger a workflow at a scheduled time.
> > You can schedule a workflow to run at specific UTC times using [POSIX cron syntax]> (https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07). Scheduled workflows run on the latest commit on the default or base branch. The shortest interval you can run scheduled workflows is once every 5 minutes.

Yes, I think this would be useful.

**Review by AkagiShigeru — APPROVED** [review]
Reviewed and tested. Some minor additions and fixes have been pushed.
Working:

- Unit tests
- Integration tests
- Menu creation
- Grocery list creation
- Todoist calls

The code looks good, thanks :).

**Pull request merged**